src/arrayMethodJoin.js

```diff
@@ -5,8 +5,23 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
Перше, можна поставити значення за замовчуванням, напевне найоптимальніший вар...
```suggestion
  [].__proto__.join2 = function(separator = ',') {
```